src/js/components/ContentInfo/index.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import React from 'react';
+import FilmItem from '../FilmItem';
+
+type Props = {
+ onFilmSelect: (id: number) => void,
+ films: []
+}
+
+function filmItems(props: Props) {
+  if (props.films && props.films.length) {
+    return props.films.map((item, ndx) => (
+      <li
+        className='film-list__item'
+        key={item.id}
+        >
+        <FilmItem
+          film={item}
+          id={item.id}
+          onFilmSelect={props.onFilmSelect}
+        />
+      </li>
+    ));
+  }
+  return <h2 className='no-data'>No films found</h2>;
+}
+
+const ContentInfo = (props: Props) => (
+  <div className='content-info'>
```

Comment:
To me, this whole component looks like `FilmItems`, so I would create a separate `FilmItem` component for:
```jsx
<li
 +        className='film-list__item'
 +        key={item.id}
 +        >
 +        <FilmItem
 +          film={item}
 +          id={item.id}
 +          onFilmSelect={props.onFilmSelect}
 +        />
 +      </li>
```

and then change `ContentInfo` to:
```jsx
    ....
    <ul ...>
        {props.films.length ? (
             props.films.map(film => <FilmItem {...film} onFilmSelect={props.onFilmSelect} />);
        ) : (
             <h2 className='no-data'>No films found</h2>
        )}
    </ul>
```

What do you think about it?